mod_reforged/hooks/skills/actives/taunt.nut

```diff
@@ -1,8 +1,48 @@
 ::Reforged.HooksMod.hook("scripts/skills/actives/taunt", function(q) {
+	q.m.DefenseModifierFraction <- 0.2;	// This percentage of the users resolve is removed from the targets defense
+	q.m.MaxRangeDefenseDebuff <- 1;
+
+	q.getTooltip = @(__original) function()
+	{
+		local ret = __original();
+
+		if (this.m.DefenseModifierFraction != 0)
+		{
+			ret.push({
+				id = 12,
+				type = "text",
+				icon = "ui/icons/special.png",
+				text = "When used on an adjacent target, reduce its Melee Defense and Ranged Defense by " + ::MSU.Text.colorizeFraction(this.m.DefenseModifierFraction) + " of your current Resolve (" + ::MSU.Text.colorizeValue(this.calculateDefenseModifier()) + ")"
```

Comment:
I like that you show the actual modifier that will be applied. But it is slightly confusing to place it at the end in parantheses after `current Resolve` as it can be taken to mean that this value represents your current resolve. Maybe to clarify you could add the word `by` before it e.g. `(by x)`. Or you can say it like: "When used on an adjacent target, reduce its Melee Defense and Ranged Defense by X (20% of your current Resolve)." so the actual value is shown first and its explanation that it is 20% of your resolve is given in the parantheses. What are your thoughts?

Reply:
I will use your last suggestion